egs2/chime2/enh1/local/find_noisy_transcripts.pl

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env perl
```

Comment:
Ditto.

Reply:
Yes, now I reused the one from egs/chime4/asr1.